pipeline_dp/combiners.py

```diff
@@ -208,3 +187,29 @@ def compute_metrics(
         for combiner, acc in zip(self._combiners, accumulator):
             metrics.append(combiner.compute_metrics(acc))
         return (privacy_id_count, metrics)
+
+
+def create_compound_combiners(
```

Comment:
create_compound_combiner (without "s")? It looks like the method returns a single compound combiner (even if there are multiple child combiners)

Reply:
Thanks, correct!